app/models/ticket.rb

```diff
@@ -1,4 +1,9 @@
 class Ticket < ActiveRecord::Base
   belongs_to :app
   has_many :acceptance_criteria
+  belongs_to :owner, class_name: "User"
+
+  def branch_name
+    "#{owner.nickname}/#{id}/#{description.parameterize}"
```

Comment:
I'd think that `description` could get a bit long. Let's wait until we have a `title` attribute and use that here.